ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -2411,6 +2415,40 @@ public sealed class Commands {
 			return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 		}
 
+		private async Task<string?> ResponsePointBalance(ulong steamID) {
+			if ((steamID == 0) || !new SteamID(steamID).IsIndividualAccount) {
+				throw new ArgumentOutOfRangeException(nameof(steamID));
+			}
+
+			long? myPoint = await Bot.ArchiWebHandler.GetPointBalance().ConfigureAwait(false);
+
+			return !Bot.IsConnectedAndLoggedOn ? FormatBotResponse(Strings.BotNotConnected) : FormatBotResponse(string.Format(CultureInfo.CurrentCulture, Strings.BotPointBalance, myPoint));
```

Comment:
This is unacceptable, you should check if bot is connected firstly and send the request only if it is.